crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -730,14 +713,16 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
                 let TableDef {
                     table_name,
                     columns,
+                    indexes: _,
+                    constraints: _,
+                    sequences: _,
```

Comment:
As discussed, I think we need reject updates unless the tables are exactly equal. That is compare `indexes`, `constraints` and `sequences` too (sorted!).

Reply:
I pushed the change